reconstruction/rtpc/src/main/java/org/jlab/rec/rtpc/KalmanFilter/KFitter.java

```diff
@@ -61,7 +61,7 @@ public void predict(Indicator indicator) throws Exception {
 		double E = Math.sqrt(p * p + mass * mass);
 		double dE = Math.abs(stepper.dEdx);
 
-		double K = 0.307075;
+		double K = 0.000307075;
```

Comment:
Since you are using MeV and cm, the correct value for the K factor should be 0.307075 MeV g-1 cm2.
Was the problem K or something else? what about the *1000*1000?

Reply:
indicator.material.getDensity() * K * indicator.material.getZoverA() / beta2 * tmax * s / 10 * (1.0 - beta2 / 2) this part compute sigma_dE_square in GeV^2 with K = 0.0003... and s in mm. 

After that I need  the sigma_dE_square in MeV so I have a factor 10^6